docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -41,166 +39,251 @@ This function returns the `fixture` object that allows you to configure the fixt
 
 > [Tests](#tests) that constitute a fixture go after this declaration.
 
-### Specifying the Start Webpage
+## Tests
 
-You can optionally use the fixture declaration to specify a webpage at which all tests in a fixture start.
-To do this, use the `page` function.
+To introduce a test, call the `test` function and pass test code inside.
 
 ```text
-page( url )
+test( testName, fn(t) )
+```
+
+Parameter  | Type     | Description
+---------- | -------- | --------------------------------------------------------------------
+`testName` | String   | The test name.
+`fn`       | Function | An asynchronous function that contains test code.
+`t`        | Object   | The [test controller](#test-controller) used to access test run API.
+
+```js
+fixture `MyFixture`;
+
+test('Test1', async t => {
+    /* Test 1 Code */
+});
+
+test('Test2', async t => {
+    /* Test 2 Code */
+});
 ```
 
+You are free to structure test code in any manner, and you can
+reference any modules or libraries you need.
+
+TestCafe tests are executed server side. To manipulate the tested webpage, use [test actions](actions/index.md).
+To determine the state of page elements or obtain any other data from the client side, use [selectors](selecting-page-elements/selectors.md) and
+[client functions](obtaining-data-from-the-client.md).
+
+To check if the page state matches the expected one, use [assertions](assertions/index.md).
+
+### Test Controller
+
+A *test controller* object `t` exposes methods of test API. That is why it is
+passed to each function that is expected to contain server-side test code (like [test](#tests),
+[beforeEach](#initialization-and-clean-up) or [afterEach](#initialization-and-clean-up)).
+
+Use the test controller to call [test actions](actions/index.md), handle [browser dialogs](handling-native-dialogs.md),
```

Comment:
> And what about to include eval in a-z index for Test Controller?

`eval` is already there.

>  we want to list all for test controller we should mention eval and ctx

1) this part is old - not added recently. diffs are wrong
2) I didn't mean to list everything test controller is used for